tests/functional/codegen/calling_convention/test_external_contract_calls.py

```diff
@@ -2496,16 +2497,16 @@ def do_stuff(f: Foo) -> uint256:
 
 
 @pytest.mark.parametrize("typ,val", [("address", TEST_ADDR)])
-def test_calldata_clamp(w3, get_contract, tx_failed, keccak, typ, val):
+def test_calldata_clamp(w3, get_contract, tx_failed, typ, val):
     code = f"""
 @external
 def foo(a: {typ}):
     pass
     """
     c1 = get_contract(code)
-    sig = keccak(f"foo({typ})".encode()).hex()[:10]
+    sig = method_id(f"foo({typ})").hex()
```

Comment:
note: we should have more unit tests for `method_id` util (but out of scope for this PR i think)